src/libraries/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/ValidationAttribute.cs

```diff
@@ -273,8 +273,6 @@ private void SetResourceAccessorByPropertyLookup()
                                                     _errorMessageResourceType.FullName));
             }
 
-
-            // TODO-NULLABLE: If the user-provided resource returns null, an ArgumentNullException is thrown - should probably throw a better exception
```

Comment:
Replaced with https://github.com/dotnet/runtime/issues/44292

Reply:
(this will likely be closed so I decided to remove it here but allow owner to make final call)